nextcloud/README.md

```diff
@@ -21,10 +21,11 @@
 - Environment variables provided (see below).
 
 ### Tags
-- **latest** : latest stable version (22.1)
-- **22.1** : latest 22.1.x version (stable)
-- **21.0** : latest 21.0.x version (old stable)
-- **20.0** : latest 20.0.x version (deprecated)
+- **latest** : latest stable version (23.0)
+- **23.0** : latest 23.0.x version (stable)
+- **22.1** : latest 22.1.x version (old stable)
```

Comment:
This should be 22.2, see contents of https://github.com/hoellen/dockerfiles/tree/master/nextcloud

Reply:
Jepp ... I just added the new version and EOLed the old version changed the "release names". I added a new commit fixing this.